friends/src/App.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import styled from 'styled-components';
+import { Button, FormGroup, Form, Input, Tooltip, Row, Col, Container } from 'reactstrap';
+
+import { FriendsList } from './components/FriendsList';
+
+const FormContainer = styled.div`
+display: flex;
+flex-direction: column;
+align-items: flex-start;
+margin-top: 50px;
+`
+
+export default class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friendsList: [],
+      tooltipOpen: false,
+      name: "",
+      age: "",
+      email: ""
+    }
+  }
+
+  toggleToolTip = () => {
+    this.setState({
+    tooltipOpen: !this.state.tooltipOpen
+    });
+}
+
+  componentDidMount() {
+    axios.get('http://localhost:5000/friends').then(response => this.setState({friendsList: [...response.data]}))
+    .catch(err => console.error(err))
+  }  
+
+  addNewFriend = e => {
+      e.preventDefault();
+      if(this.state.name && this.state.age && this.state.email) {
+          const friend = {
+          id: this.state.friendsList.length + 1,
```

Comment:
Is setting an id necessary? The server.js provides an id and returns the entire friends object back.